cif_pow.dic

```diff
@@ -6211,27 +6211,27 @@ save_pd_pref_orient_sphericalharmonics.c_ij
 
 save_
 
-save_pd_pref_orient_sphericalharmonics.c_ij_su
+save_pd_pref_orient_spherical_harmonics.c_ij_su
 
-    _definition.id                '_pd_pref_orient_sphericalharmonics.c_ij_su'
+    _definition.id                '_pd_pref_orient_spherical_harmonics.c_ij_su'
     _definition.update            2022-11-17
     _description.text
 ;
-    Standard uncertainty of _pd_pref_orient.sphericalharmonics_c_ij_su.
+    Standard uncertainty of _pd_pref_orient.spherical_harmonics_c_ij_su.
```

Comment:
```suggestion
    Standard uncertainty of _pd_pref_orient_spherical_harmonics.c_ij.
```

Reply:
I don't understand? That change is already made in my branch? Oh well. Committed.

Reply:
After writing up the suggestion I noticed that there is an option to commit to this PR branch and made the actual change. Sorry for the confusion.